binr/rafind2/rafind2.c

```diff
@@ -76,12 +76,12 @@ static int hit(RSearchKeyword *kw, void *user, ut64 addr) {
 }
 
 static int show_help(char *argv0, int line) {
-	printf ("Usage: %s [-mXnzZhv] [-a align] [-b sz] [-f/t from/to] [-[m|s|S|e] str] [-x hex] file ..\n", argv0);
+	printf ("Usage: %s [-mXnzZhv] [-a align] [-b sz] [-f/t from/to] [-[e|s|S] str] [-x hex] file ..\n", argv0);
 	if (line) return 0;
 	printf (
 	" -a [align] only accept aligned hits\n"
 	" -b [size]  set block size\n"
-	" -e [regex] search for regular expression string matches\n"
+	" -e [regex] search for regular expression string matches(can be used multiple times)\n"
```

Comment:
Use regex instead of the long name and add a space before the (. This msg is suposed to fit in 80 columns